include/rocksdb/sst_file_writer.h

```diff
@@ -128,6 +128,8 @@ class SstFileWriter {
   // REQUIRES: user_key is after any previously added point (Put/Merge/Delete)
   //           key according to the comparator.
   // REQUIRES: timestamp's size is equal to what is expected by the comparator.
+  // When Options.persist_user_defined_timestamps is set to false, the timestamp
+  // part will not be included in SST file.
```

Comment:
We could also mention that the min timestamp has to be used in this case.

Reply:
Thank you for catching this! I updated the documentation.